pkg/providers/aws/config.go

```diff
@@ -184,11 +184,7 @@ func CreateSessionFromStrategy(ctx context.Context, c client.Client, credentials
 	} else {
 		awsConfig.Credentials = awsCredentials.NewStaticCredentials(credentials.AccessKeyID, credentials.SecretAccessKey, "")
 	}
-
-	sess, err := session.NewSession(&awsConfig)
-	if err != nil {
-		return nil, errorUtil.Wrapf(err, "failed to create aws session from strategy, region=%s", region)
-	}
+	sess := session.Must(session.NewSession(&awsConfig))
```

Comment:
I'm not sure about this change. The error is handled by panicing rather than reporting an error back. I beleive this pattern makes it more difficult to debug issues.

Reply:
https://docs.aws.amazon.com/sdk-for-go/api/aws/session/#Must

The `Must` helper function seems to be widely used in AWS SDK code examples.
I agree it may cause debugging to be slightly more difficult, but it saves us a refactor to include this error check in code coverage. This file is the only place in CRO where we're initializing new sessions, so maybe it won't be that difficult to debug. I can change that piece to handle the error in our code, if requested.